### DISCUSSION TITLE
Error While Importing 1080i  Videos

### DISCUSSION THREAD

**Discussion by donnafaye0323** [justdan96/tsMuxer#770] [Enhancements]
Hello, I imported the two videos of 1080i 60fps from my USB drive and it said "tsmuxer error". This doesn't see right. Would you help fix this issue? I hope it works out.

#700

**Comment by jcdr428** [comment]
@donnafaye0323 can you please provide a link to download a sample.

**Comment by DreckSoft** [comment]
I provided a sample in #700

**Comment by jcdr428** [comment]
Closing, issue #700 has been fixed.
